packages/protocol/test/identity/attestations.ts

```diff
@@ -566,15 +624,20 @@ contract('Attestations', (accounts: string[]) => {
       assert.equal(numTotal.toNumber(), attestationsRequested)
     })
 
-    it('should set the block number of the successful completion', async () => {
+    it('should set the time of the successful completion', async () => {
+      await sleep(1000)
```

Comment:
Why is this necessary?

Reply:
In the original timestamp setup, I wanted the time to change so that it's clear that we are recording the completion timestamp, but now with blocks this becomes 1) invalid and 2) slightly less important. Changing it to `advanceBlockNum`